packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,82 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    &__grid {
+      @include carbon--make-container;
+    }
+    &__row {
+      @include carbon--make-row;
+    }
+    &__left,
+    &__children {
+      @include carbon--make-col-ready;
+    }
+    &__left {
+      @include carbon--breakpoint(lg) {
+        @include carbon--make-col(4, 16);
+      }
+    }
+    &__heading {
+      margin-bottom: $carbon--layout-05;
+
+      @include type-style('expressive-heading-02');
+
+      @include carbon--breakpoint('sm') {
+        margin-top: $carbon--layout-01;
+      }
+      @include carbon--breakpoint('lg') {
+        margin-top: $carbon--layout-03;
+      }
```

Comment:
This is adding too much spacing. I don't see this in the specs.

Reply:
So can I reduce the margin or remove totally?

Reply:
If I'm reading it right this needs to be removed entirely. you should already have `$carbon--layout-01` which is all you need I think.

Reply:
Okay